service/src/main/java/org/databiosphere/workspacedataservice/SentryInitializer.java

```diff
@@ -89,24 +83,12 @@ Map<String, String> getTags() {
     return tags;
   }
 
-  /**
-   * Extracts an environment (e.g. "dev" or "prod") from a Sam url. Looks for ".dsde-${env} and
-   * returns ${env} if found. Also looks for BEEs and returns the bee name if found. Else, returns
-   * "unknown".
-   *
-   * @param samUrl the url to Sam
-   * @return the environment as parsed from the Sam url
-   */
-  protected String urlToEnv(String samUrl) {
-    if (samUrl == null) {
-      return DEFAULT_ENV;
-    }
-    Matcher matcher = SAM_ENV_PATTERN.matcher(samUrl);
-    boolean found = matcher.find();
-    if (found) {
-      return matcher.group(1);
-    } else {
-      return DEFAULT_ENV;
+  boolean determineIfEnvIsMonitored(String env) {
```

Comment:
Super nit, but a few alternative suggestions for function names: 

```suggestion
  boolean isEnvMonitored(String env) {
```
or
```suggestion
  boolean shouldMonitorEnv(String env) {
```

Reply:
Alternatively, since the `env` passed is stored in a property of this class, does this method need to take an argument?

Reply:
env only exists in the context of the constructor at this time.